forge-ai/src/main/java/forge/ai/ComputerUtilCombat.java

```diff
@@ -2044,14 +2045,8 @@ public static Map<Card, Integer> distributeAIDamage(final Player self, final Car
         boolean isAttackingMe = isAttacking && combat.getDefenderPlayerByAttacker(attacker).equals(self);
         boolean isBlockingMyBand = false;
         if (attacker.getController().isOpponentOf(self)) {
-            List<String> bandsWithString = Arrays.asList("Bands with Other Legendary Creatures",
-                    "Bands with Other Creatures named Wolves of the Hunt",
-                    "Bands with Other Dinosaurs");
-            for (final Card c : block) {
-                if (c.hasKeyword(Keyword.BANDING) || c.hasAnyKeyword(bandsWithString)) {
-                    isBlockingMyBand = true;
-                    break;
-                }
+            if (AttackingBand.isValidBand((List<Card>)block, true)) {
```

Comment:
I'm unsure if we can trust this cast or if we should make a new Array List?

Reply:
I was also thinking about it :) Debugging that place shows that it converts cleanly, but I dunno :) I guess let's make a new ArrayList instead. I was thinking if there's a cleaner/nicer way to get a list from a CardCollection, but couldn't come up with anything except for block.subList(0, block.size()) (which works too, I guess)

Reply:
Or make isValidBand get a java Collection too , it shouldn't try to alter it anyway so it shouldn't be a problem?

Reply:
Yeah, good idea. Done :)